src/components/backend-ai-usersettings-general-list.ts

```diff
@@ -800,6 +1008,60 @@ export default class BackendAiUsersettingsGeneralList extends BackendAIPage {
                      ${_t("button.Save")}</wl-button>
         </div>
       </backend-ai-dialog>
+      <backend-ai-dialog id="ssh-keypair-management-dialog" fixed backdrop persistent>
+        <span slot="title">${_t("usersettings.SSHKeypairManagement")}</span>
+          <div slot="content" style="max-width:500px">
+            <span slot="title"> ${_t("usersettings.CurrentSSHPublicKey")}</span>
+            <mwc-textarea class="ssh-keypair" style="width:450px;" id="current-ssh-public-key" outlined readonly></mwc-textarea>
+            <wl-button class="copy" @click="${() => this._copySSHKey("#current-ssh-public-key")}">
+              <wl-icon>content_copy</wl-icon>
+            </wl-button>
+          </div>
+          <div slot="footer">
+            <wl-button class="cancel" inverted flat @click="${this._hideSSHKeypairDialog}">${_t("button.Cancel")}</wl-button>
+            <wl-button class="ok" @click="${this._openSSHKeypairGenerationDialog}">${_t("button.Generate")}</wl-button>
+          </div>
```

Comment:
The first `span` element and the following two `div` elements should be aligned with the same indent.